SearchRotatedSorted.java

```diff
@@ -0,0 +1,47 @@
+// Time Complexity : O(log n)
```

Comment:
what is n?

Reply:
Time complexity: O(log mn) where m is a number of rows and n is a number of columns of a given matrix.

Reply:
correct